sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala

```diff
@@ -139,7 +140,13 @@ object ReloadWorkflow {
       )
     IO.write(entryPoint, depsFileContent.show)
 
-    Webpack.run(entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
+    customWebpackConfigFile match {
+      case Some(configFile) =>
+        Webpack.run("--config", configFile.getAbsolutePath, entryPoint.absolutePath, bundleFile.absolutePath)(workingDir, logger)
```

Comment:
Actually i meant a setting for the module config file that would replace the "webpackConfigFile in reloadWorkflow" i use now. This file can than be copied to target and used in reloadWorkflow and included in the generated webpack config (using require and maybe some js for merging the settings). Because it is copied it can also be included in a custom config using a relative filename.